executor/executor.go

```diff
@@ -454,8 +481,8 @@ func (executor *Executor) getServiceDetails(serviceID uuid.UUID) (namespace, nam
 	return
 }
 
-func (executor *Executor) getContainerDetails(containerID uuid.UUID) (name string, err error) {
-	name, ok := executor.scanner.FindContainerNameByID(executor.scanner.GetApplications(), containerID)
+func (executor *Executor) getContainerDetails(containerID uuid.UUID) (container *scanner.Container, service *scanner.Service, app *scanner.Application, err error) {
```

Comment:
method name: `getContainerDetails` is confusing here, why do we return `service` and `app` as separate values?

Go style is to return only one value and error (if needed). if you need multiple values, create `struct` to group them in this case `containerDetails`.

Reply:
updated